kim_cnn/network/cnnTextNetwork.py

```diff
@@ -152,6 +155,17 @@ def train(self):
 
       epoch += 1
       accuracy = float(acc_corrects) / float(acc_sents) * 100
+
+      # if the new accuracy is better than the old accuracy by 1e-3
+      if accuracy - best_accuracy > 0.01:
```

Comment:
1e-3 != 0.01, I suspect you meant 0.001?

Reply:
Sorry, it should be 0.1 since I meant it in terms of percentage. I'll update the PR and the comment.